<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>672</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Sam Rocket!</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-40</y>
      <width>381</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>part2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>691</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>1011</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>431</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Get Data</string>
    </property>
    <widget class="QPushButton" name="getHead">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>391</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Get Data Head</string>
     </property>
    </widget>
    <widget class="QPushButton" name="getCSV">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>391</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Get Original CSV</string>
     </property>
    </widget>
    <widget class="QPushButton" name="getTruncated">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>391</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Get Truncated CSV</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>431</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QPushButton" name="openFiles">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>391</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Open Files</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showData">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>391</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Display Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="getParams">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>391</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Show With Parameters</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-40</y>
      <width>361</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo (2).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
